src/test/config/storage.test.ts

```diff
@@ -0,0 +1,50 @@
+import * as assert from "assert";
+import * as TypeMoq from "typemoq";
+import * as vscode from "vscode";
+
+import Storage from "../../config/storage";
+
+suite("Storage tests", () => {
+
+    const createMocks = () => {
```

Comment:
Arrow syntax used for a named function here, I believe they were to be reserved for anonymous functions for now

Reply:
Fair enough. This was the style used in default tests so I followed it in the absence of existing examples.

Reply:
The current setup of TSLint complains if these arrows are turned into function syntax.